.idea/misc.xml

```diff
@@ -5,7 +5,6 @@
       <configuration PROFILE_NAME="Debug" CONFIG_NAME="Debug" />
     </configurations>
   </component>
-  <component name="ExternalStorageConfigurationManager" enabled="true" />
```

Comment:
Is this change part of the pull request intentionally? Is it somehow related to implementation?

Reply:
No, It shouldn't be, let me fix the conflicts